openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
Pointing to other languages' features won't change my mind. :) This doesn't make any sense in Python. You're the first person I've seen do it, and it will confuse everyone.